src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/autocrafters/AbstractAutoCrafter.java

```diff
@@ -391,6 +392,29 @@ protected boolean matchesAny(Inventory inv, Map<Integer, Integer> itemQuantities
         return false;
     }
 
+    public boolean inventoryAvailable(@Nonnull Inventory inv, @Nonnull ItemStack resultItem) {
+
+        if(inv.firstEmpty() < 0){
+
+            for (ItemStack matchingItem : inv.getContents()) {
+                // Checks if any item stack that is not full that is the same type as the result.
+                if (
+                        ((!(resultItem instanceof SlimefunItemStack) && resultItem.isSimilar(matchingItem))
+                        || ((resultItem instanceof SlimefunItemStack) && SlimefunUtils.isItemSimilar(matchingItem, resultItem, true)))
+                        && matchingItem.getAmount() < matchingItem.getMaxStackSize()
+                ){
```

Comment:
formatting :notLikeThis:

Reply:
Ops, sorry. I read the code style guidelines, I just didn't know what does line-wrapping means.